packages/html/src/plugins/responsive.ts

```diff
@@ -12,10 +12,9 @@ import {isImage} from "../utils/isImage";
  * @description Updates the src with the size of the parent element and triggers a resize event for
  * subsequent resizing.
  * @param steps {number | number[]} The step size in pixels.
```

Comment:
```suggestion
 * @param steps {number | number[]} The step size in pixels or an array of image widths in pixels.
```

Reply:
Based on your description in Slack, I think this is correct. They are image widths.